Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml

```diff
@@ -39,7 +39,7 @@ configuration:
   - D - Not usually reliable
   - E - Unreliable
   - F - Reliability cannot be judged
-  required: false
+  required: True
```

Comment:
?

Reply:
This field should be `True`, it wasn't so changed it.
Also, verified it with @avidan-H